app/src/main/res/raw/sessions_en.json

```diff
@@ -38,11 +38,11 @@
     "etime": "2016-02-18 12:00:00",
     "category": {
       "id": 2,
-      "name": "その他"
+      "name": "Others"
     },
     "place": {
       "id": 2,
-      "name": "ルームA"
+      "name": "Hole A"
```

Comment:
Ah, I noticed our website uses "Room" instead of "Hole". And it calls "Keynote Venue" as "Room A". So confusing :confused: